libc/functions/g/getpgid.part-impl.md

```diff
@@ -0,0 +1,47 @@
+# Synopsis 
+`#include <unistd.h>`</br>
+
+` pid_t getpgid(pid_t pid);`</br>
+
+## Status
+Partially implemented
+## Conformance
+IEEE Std 1003.1-2017
+## Description
+
+
+The `getpgid()` function shall return the process group ID of the process whose process ID is equal to _pid_. If _pid_ is equal to `0`, `getpgid()` shall return the process group ID of the calling process.
+
+
+## Return value
+
+
+Upon successful completion, `getpgid()` shall return a process group ID. Otherwise, it shall return `(pid_t)-1` and set _errno_ to indicate the error.
```

Comment:
```suggestion
Upon successful completion, `getpgid()` shall return a process group ID. Otherwise, it shall return `(pid_t)-1` and set `errno` to indicate the error.
```